search-app/src/cmr/search/site/data.clj

```diff
@@ -35,11 +35,11 @@
   request context which contains the state of a running CMR."
   :execution-context)
 
-;; (defmethod get-providers :cli
-;;   [context]
-;;   (let [providers-url (format "%sproviders"
-;;                               (config/application-public-root-url :ingest))]
-;;     (util/endpoint-get providers-url {:accept mt/json})))
+(defmethod get-providers :cli
```

Comment:
Delete this code that is no longer used.

Reply:
It's still used here: `cmr.search.site.static` when calling `lein run -m cmr.search.site.static static-site`. I use this for local dev debugging.